### PR TITLE
Feature/sync profile with bigquery

### PR DESCRIPTION
**Pull request by siddhant3030**
## Summary
 This is to sync profiles table to bigquery

## Checklist
  Before submitting a pull request, please ensure that you mark these task.

- [ ] Ran `mix setup` in the repository root and test.
- [ ] If you've fixed a bug or added code that is tested and has test cases. 
- [ ] In the case of adding a new API, you added a **postman** request for that.
- [ ] Coding standard and conventions are followed. https://docs.google.com/document/d/1rfU33IjS-ioiIH0TBWpxoLsdyyYdI1tDrqr5HCRIb98/edit?usp=sharing

## Notes
 Please add here if any other information is required for the reviewer.

### CODE REVIEW

**Comment by AkhileshNegi** [comment]
@siddhant3030 can you share what this PR does? As per my testing this will just create a table in in bigquery dataset but data also needs to be synced.
Can you share steps you used to test this PR

**Comment by siddhant3030** [comment]
> @siddhant3030 can you share what this PR does? As per my testing this will just create a table in in bigquery dataset but data also needs to be synced. Can you share steps you used to test this PR

How do I sync the data? Can you share some examples of contact table?

**Comment by siddhant3030** [comment]
> > @siddhant3030 can you share what this PR does? As per my testing this will just create a table in in bigquery dataset but data also needs to be synced. Can you share steps you used to test this PR
> 
> How do I sync the data? Can you share some examples of contact table?

File: lib/glific/third_party/bigquery/bigquery_worker.ex [hunk 1]
```diff
@@ -253,6 +260,44 @@ defmodule Glific.BigQuery.BigQueryWorker do
     :ok
   end
 
+  defp queue_table_data("profiles", organization_id, attrs) do
+    # This function will fetch all the profiles from the database and will insert it in bigquery in some chunks.
```
Comment (AkhileshNegi) [inline lib/glific/third_party/bigquery/bigquery_worker.ex]: This function will fetch all the profiles from the database and will insert it in bigquery in chunks 0f 100

**Review by AkhileshNegi — APPROVED** [review]

**Pull request merged**